tutor/templates/build/openedx/Dockerfile

```diff
@@ -265,6 +265,8 @@ RUN --mount=type=cache,target=/openedx/.cache/pip,sharing=shared \
 # https://pypi.org/project/ipython (>=Python 3.10 started with 8.20)
 RUN --mount=type=cache,target=/openedx/.cache/pip,sharing=shared \
     pip install ipdb==0.13.13 ipython==8.24.0
+# Re-install forked edx-search which is otherwise overridden by development.txt
+RUN pip install git+https://github.com/regisb/edx-search@regisb/meilisearch#edx-search
```

Comment:
nit: this can be removed now that [edx-search==4.1.0](https://github.com/openedx/edx-search/tree/v4.1.0) is now installed in [`master`](https://github.com/openedx/edx-platform/blob/master/requirements/edx/base.txt#L516-L517) and [`open-release/sumac.master`](https://github.com/openedx/edx-platform/blob/open-release/sumac.master/requirements/edx/base.txt#L516-L517).

Reply:
:+1: